src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/steps/DecisionSteps.java

```diff
@@ -212,4 +244,52 @@ public void theDecisionExpiryDateEmpty() throws Throwable {
         DecisionResponse decision = JsonUtils.toObjectFromJson(testContext.getHttpContext().getRawResponseString(), DecisionResponse.class);
         assertNull(decision.getDeadlineExpiryDate());
     }
+
+    @And("^the decision replies list contains (\\d+) decision replies$")
+    public void theDecisionRepliesListContainsDecisionReplies(int expectedDecisionReplies) {
+        AllDecisionRepliesResponse allDecisionRepliesResponse =
+                testContext.getScenarioContext().getAllDecisionRepliesResponse();
+
+        int n = 0;
+        for (DecisionReply expectedDecisionReply : testContext.getScenarioContext().getDecisionReplies()) {
+            DecisionReplyResponse decisionReplyResponse = allDecisionRepliesResponse.getDecisionReplyList().get(n);
+
+            assertNotNull(decisionReplyResponse);
+
+            assertEquals(expectedDecisionReply.getDecisionReplyReason(),
+                    decisionReplyResponse.getDecisionReplyReason());
+
+            String replyState =
+                    expectedDecisionReply.getDecisionReply()
+                            ? DecisionsStates.DECISIONS_ACCEPTED.getStateName()
+                            : DecisionsStates.DECISIONS_REJECTED.getStateName();
+
+            assertEquals(replyState, decisionReplyResponse.getDecisionReply());
+
+            assertEquals(expectedDecisionReply.getDecision().getDecisionId().toString(),
+                    decisionReplyResponse.getDecisionId());
+
+            assertEquals(expectedDecisionReply.getId().toString(), decisionReplyResponse.getDecisionReplyId());
+
+            assertEquals(expectedDecisionReply.getAuthorReferenceId(), decisionReplyResponse.getAuthorReference());
+
+            n++;
+        }
+
+        assertEquals(expectedDecisionReplies, n);
+    }
+
+    @And("^the decision reply contains all the fields$")
+    public void theDecisionReplyContainsAllTheFields() {
+        DecisionReplyResponse decisionReplyResponse = testContext.getScenarioContext().getDecisionReplyResponse();
+        DecisionReply expectedDecisionReply = testContext.getScenarioContext().getDecisionReplies().stream()
+                                                .filter(dr -> dr.getId().toString().equalsIgnoreCase(decisionReplyResponse.getDecisionReplyId()))
+                                                .findFirst()
+                                                .get();
```

Comment:
Calling `get()` without `isPresent()` check may cause NPE. Either way the test will fail, nothing crucial.